app/views/toilets/_edit_toilet_form.html.erb

```diff
@@ -0,0 +1,54 @@
+
+<%= simple_form_for toilets_path(@toilet), method: :patch do |f| %>
+
+  <div class="form-inputs">
+  <div class="form-inputs">
+    <div class="input-field">
+      <%= f.input :name,
+                  label: false,
+                  placeholder: "Name" %>
```

Comment:
Capitalized 'name' as requested

Reply:
I appreciate it. Looking forward to future work with you!